docs/reference/search/search-your-data/semantic-text-hybrid-search

```diff
@@ -0,0 +1,197 @@
+[[semantic-text-hybrid-search]]
+=== Tutorial: hybrid search with `semantic_text`
+++++
+<titleabbrev>Hybrid search with `semantic_text`</titleabbrev>
+++++
+
+This tutorial demonstrates how to perform **hybrid search**, combining semantic search with traditional full-text search.
+
+In hybrid search, semantic search retrieves results based on the meaning of the text, while full-text search focuses on exact word matches. By combining both methods, hybrid search delivers more relevant results, particularly in cases where relying on a single approach may not be sufficient.
+
+The recommended way to use hybrid search in the {stack} is following the `semantic_text` workflow. This tutorial uses the <<inference-example-elser,`elser` service>> for demonstration, but you can use any service and its supported models offered by the {infer-cap} API.
+
+[discrete]
+[[semantic-text-hybrid-infer-endpoint]]
+==== Create the {infer} endpoint
+
+Create an inference endpoint by using the <<put-inference-api>>:
```

Comment:
From 8.16, the default service for `semantic_text` is ELSER. You don't need to create an inference endpoint first, you just create an index with a semantic text field without specifying an inference_id and an inference endpoint with the ELSER service will be automagically created. This workflow will be demonstrated in the original semantic_text tutorial.

It is valuable to have the inference endpoint creation step documented here even if the example uses the default endpoint, because users can see what should be done in case they use a different service. However, we should mention that the default service is ELSER and we include this step only for demonstrating how the endpoint creation works.